x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_management/maintenance_windows/maintenance_window_callout.cy.ts

```diff
@@ -12,9 +12,10 @@ import { login } from '../../../../tasks/login';
 import { visit } from '../../../../tasks/navigation';
 import { RULES_MANAGEMENT_URL } from '../../../../urls/rules_management';
 
+// https://github.com/elastic/kibana/issues/184160
 describe(
   'Maintenance window callout on Rule Management page',
-  { tags: ['@ess', '@serverless'] },
+  { tags: ['@ess', '@skipInServerless'] },
```

Comment:
Should it be:

```suggestion
  { tags: ['@ess', '@serverless', '@skipInServerless'] },
```

Also, are we changing the roles only for MKI executions, or for both MKI and CI executions? If the former, let's skip the test only in MKI:

```suggestion
  { tags: ['@ess', '@serverless', '@skipInServerlessMKI'] },
```

Reply:
We are changing the default user for both stateless and MKI so I'm keeping your first suggestion. Thanks!! :)